tidb-cloud/serverless-driver.md

```diff
@@ -59,6 +59,33 @@ try {
 }
 ```
 
+### Custom column type decoders
```

Comment:
I think we can put this example in the `Configure the serverless driver` section